src/Http/Controllers/Validators/Relations.php

```diff
@@ -30,15 +30,23 @@ public static function validate(Controller $controller, array $relations)
             return $relations->all();
         }
 
-        $relationsAreValid = $relations->every(function ($callback, $relation) use ($availableRelations) {
+        $invalidRelations = [];
+
+        $relations->each(function ($callback, $relation) use ($availableRelations, &$invalidRelations) {
             $key = is_string($callback) ? $callback : $relation;
 
-            return array_key_exists($key, $availableRelations) || in_array($key, $availableRelations);
+            if (! array_key_exists($key, $availableRelations) && ! in_array($key, $availableRelations)) {
+                $invalidRelations[] = $key;
+            }
         });
 
-        throw_if(! $relationsAreValid, ValidationException::withMessages([
-            'relations' => 'Invalid relations'
-        ]));
+        if (! empty($invalidRelations)) {
```

Comment:
If you use the `filter` method (mentioned in the previous comment), you can do `$invalidRelations->isEmpty()` here.